Sources/Rendering/Core/AxesActor/index.js

```diff
@@ -154,11 +174,13 @@ export const DEFAULT_VALUES = {
     tipLength: 0.2,
     shaftResolution: 60,
     shaftRadius: 0.03,
-    invert: false,
```

Comment:
Done the separate config for each axis

Reply:
I agree with separated config for each axis and propably the place to put the label in the future...

Reply:
You must use `defaultValue()` instead of `DEFAULT_VALUE` (find examples in VTK.js code base).
Otherwise your configs will be shared between ALL instances.

Reply:
You changed the API of the class. Hence it is no longer a `refactor` commit, it is a BREAKING_CHANGE commit and requires a BUMP of VTK.js major version.

Reply:
Sorry, i've incorrectly interpret your review